libs/api/domains/cms/src/lib/cms.resolver.ts

```diff
@@ -298,7 +298,7 @@ export class CmsResolver {
   ): Promise<LifeEventPage[]> {
     return this.cmsContentfulService.getLifeEventsInCategory(
       input.lang,
-      input.slug,
+      input.slug ?? '',
```

Comment:
I think slug is not allowed to be `undefined` here via `class-calidator` in model. Maybe we should add a non-null assertion operator(!) to the model to reflect this, then there should be no need for check here